scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl

```diff
@@ -183,36 +187,45 @@ task ExtractFromSuperpartitionedTables {
         String dataset
         String filter_set_name
         String avro_sibling
-        Int table_index
+        Int num_superpartitions
+        Int shard_index
+        Int num_shards
     }
     parameter_meta {
         avro_sibling: "Cloud path to a file that will be the sibling to the 'avro' 'directory' under which output Avro files will be written."
-        table_index: "1-based index for the superpartitioned ref_ranges and vet tables to be extracted by this call."
+        num_superpartitions: "Total number of superpartitions requiring extraact"
+        shard_index: "0-based index of this superpartition extract shard"
+        num_shards: "Count of all superpartition extract shards"
     }
+
     command <<<
         set -o errexit -o nounset -o xtrace -o pipefail
         echo "project_id = ~{project_id}" > ~/.bigqueryrc
 
         avro_prefix="$(dirname ~{avro_sibling})/avro"
-        str_table_index=$(printf "%03d" ~{table_index})
 
-        # These bq exports error out if there are any objects at the sibling level to where output files would be written
-        # so an extra layer of `vet_${str_table_index}` is inserted here.
-        bq query --nouse_legacy_sql --project_id=~{project_id} "
-            EXPORT DATA OPTIONS(
-            uri='${avro_prefix}/vets/vet_${str_table_index}/vet_${str_table_index}_*.avro', format='AVRO', compression='SNAPPY') AS
-            SELECT location, sample_id, ref, REPLACE(alt,',<NON_REF>','') alt, call_GT as GT, call_AD as AD, call_GQ as GQ, cast(SPLIT(call_pl,',')[OFFSET(0)] as int64) as RGQ
-            FROM \`~{project_id}.~{dataset}.vet_${str_table_index}\`
-            ORDER BY location
-        "
-
-        bq query --nouse_legacy_sql --project_id=~{project_id} "
-            EXPORT DATA OPTIONS(
-            uri='${avro_prefix}/refs/ref_ranges_${str_table_index}/ref_ranges_${str_table_index}_*.avro', format='AVRO', compression='SNAPPY') AS
-            SELECT location, sample_id, length, state
-            FROM \`~{project_id}.~{dataset}.ref_ranges_${str_table_index}\`
-            ORDER BY location
-        "
+        for superpartition in $(seq ~{shard_index + 1} ~{num_shards} ~{num_superpartitions})
+        do
+            str_table_index=$(printf "%03d" $superpartition)
+
+            # These bq exports error out if there are any objects at the sibling level to where output files would be written
+            # so an extra layer of `vet_${str_table_index}` is inserted here.
+            bq query --nouse_legacy_sql --project_id=~{project_id} "
+                EXPORT DATA OPTIONS(
+                uri='${avro_prefix}/vets/vet_${str_table_index}/vet_${str_table_index}_*.avro', format='AVRO', compression='SNAPPY') AS
+                SELECT location, sample_id, ref, REPLACE(alt,',<NON_REF>','') alt, call_GT as GT, call_AD as AD, call_GQ as GQ, cast(SPLIT(call_pl,',')[OFFSET(0)] as int64) as RGQ
```

Comment:
I dont think we need to extract PLs? it doesn't look like Tim uses them. Let's ask on Wednesday. (Fine to include for now, I just happened to notice it in this pr)